src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,78 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  max-width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
```

Comment:
The `margin` property is a shorthand property for setting `margin-top`, `margin-right`, `margin-bottom`, and `margin-left` at the same time. In your case, you only need to set `margin-bottom`, so it's better to use `margin-bottom` property directly. It will make your code cleaner and easier to understand.